composer.json

```diff
@@ -27,7 +27,8 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "laminas/laminas-stdlib": "^2.7.7 || ^3.1.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "laminas/laminas-zendframework-bridge": "^1.0",
+        "webimpress/safe-writer": "^1.0"
```

Comment:
@kynx sorry for the confusion, we can use `^1.0.1 || ^2.0`. I think I've mentioned before that these have the same interface, so it should just work.

Reply:
Done

Reply:
Done